next-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="next" />
```

Comment:
is this something thats needed by nextjs?

Reply:
I think its needed for the tsc to pick up the nextjs types? So we can't remove it - At least thats what the docs say ahah

Reply:
nice nice, was just checking since i've never used nextjs before